x/auth/keeper/keeper.go

```diff
@@ -44,6 +47,10 @@ type AccountKeeperI interface {
 	GetNextAccountNumber(sdk.Context) uint64
 }
 
+type bech32Codec struct {
```

Comment:
nit: can we put this type and it's methods in a separate file? e.g. `bech32_codec.go`

Reply:
Sure!